### PR TITLE
add Class KLDivLoss and function kl_div

### PR DESCRIPTION
**Pull request by LielinJiang**
### PR types

New features
### PR changes

APIs
### Describe

添加KLDivLoss类， 并在function中添加kl_div（kldiv_loss的别名）

### CODE REVIEW

File: python/paddle/nn/layer/loss.py [hunk 1]
```diff
@@ -608,3 +611,92 @@ def forward(self, input, label):
             out = fluid.layers.reshape(out, shape=out_shape)
 
         return out
+
+
+class KLDivLoss(fluid.dygraph.Layer):
+    """
+	:alias_main: paddle.nn.KLDivLoss
```
Comment (yaoxuefeng6) [inline python/paddle/nn/layer/loss.py]: alias 相关行可以删除 会自动处理

File: python/paddle/fluid/layers/loss.py [hunk 2]
```diff
@@ -1622,47 +1622,55 @@ def huber_loss(input, label, delta):
 @templatedoc()
 def kldiv_loss(x, target, reduction='mean', name=None):
     """
-    :alias_main: paddle.nn.functional.kldiv_loss
-	:alias: paddle.nn.functional.kldiv_loss,paddle.nn.functional.loss.kldiv_loss
+    :alias_main: paddle.nn.functional.kl_div
+	:alias: paddle.nn.functional.kl_div,paddle.nn.functional.loss.kl_div
```
Comment (yaoxuefeng6) [inline python/paddle/fluid/layers/loss.py]: alias 相关行可以删除

File: python/paddle/nn/layer/loss.py (hunk 1 above)
Comment (LielinJiang) [reply python/paddle/nn/layer/loss.py]: deleted, thanks.

File: python/paddle/fluid/layers/loss.py (hunk 2 above)
Comment (LielinJiang) [reply python/paddle/fluid/layers/loss.py]: deleted, thanks

**Review by yaoxuefeng6 — APPROVED** [review]
LGTM
